src/main/java/uk/gov/ons/census/casesvc/service/RmUnInvalidateAddressService.java

```diff
@@ -33,7 +33,7 @@ public void processMessage(ResponseManagementEvent rme, OffsetDateTime messageTi
     unInvalidateAddressCase.setAddressInvalid(false);
 
     Metadata eventMetadata = null;
-    if (shouldSendCaseToField(unInvalidateAddressCase, rme.getEvent().getChannel())) {
+    if (shouldSendCaseToField(unInvalidateAddressCase)) {
```

Comment:
what happened to the channel check? seems like we lost something here - it would be impossible for shouldSendCaseToField to guess the channel, no?

Reply:
have discussed - this requirement seems to have gone away